vip-helpers/vip-migrations.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
```

Comment:
Should we use a namespace for consistency with new code? Maybe `Automattic\VIP\Migration`?

Reply:
Good idea - 1c271e3